plugins/modules/user.yml

```
@@ -172,6 +172,21 @@ DOCUMENTATION:
             - warn
           default: fail
           type: str
+        permissions_failure_action:
+          description:
+            - Controls what happens when a group specified by C(groups) is not
+              able to be modified by the user specified by C(domain_username)
+            - C(fail) is the default and will return an erro if any groups
+              membership is not modifiable by the user.
+            - C(ignore) will ignore any groups that cannot be modified.
+            - C(warn) will display a warning for any groups that cannot be
+              modified but will continue without failing.
+          choices:
+            - fail
+            - ignore
+            - warn
+          default: fail
+          type: str
```

Just need to document when this was added

```suggestion
          type: str
          version_added: 1.8.0
```